### ISSUE TITLE
Cannot reproduce result with triplet loss training?

### ISSUE THREAD

**Issue by howard5758** [jesse1029/Fake-Face-Images-Detection-Tensorflow#6]
Hi,

I used 360000 training images and 40000 val images to train a fake detection model using triplet loss (excluding pggan).
Margin = 0.8, lr = 1*e-4, epoch = 15. But the model performed quite poorly on fake images produced by pggan.
Can someone advise on the hyperparameter used to achieve 90% accuracy? Or maybe provide pretrained weights for this? Thank you very much!

Best,
PJL
